src/resolve/sourceComponent.ts

```diff
@@ -79,21 +79,6 @@ export class SourceComponent implements MetadataComponent {
     }
   }
 
-  /**
-   * Gets the metafile path of this component. Not all the types have an XML metafile,
-   * e.g., DigitalExperience has a JSON metafile (_meta.json).
-   *
-   * @returns The metafile path
-   */
-  public get metaFilePath(): string {
```

Comment:
SDR is a public API used by projects we don't own or have clear visibility to.

If you want to get rid if this, it's better to leave it and mark it deprecated, preserving functionality as it currently works.

We can delete public methods on exposed classes when we do the next major version (which is going to be pretty soon).

Reply:
Done! deprecated.